package/src/contexts/channelsStateContext/ChannelsStateContext.tsx

```diff
@@ -95,39 +81,6 @@ function reducer(state: ChannelsState, action: Action) {
         },
       };
 
-    case 'INCREASE_SUBSCRIBER_COUNT': {
```

Comment:
Hopefully this isn't used by any of our integrators (don't know if they even can).

Reply:
Sadly it was introduced with the huge offline support PR many years ago but there's no specific reference on what the intention was